packages/tools/components-package/nps.js

```diff
@@ -100,15 +100,13 @@ const getScripts = (options) => {
 			illustrations: createIllustrationsJSImportsScript,
 		},
 		copy: {
-			default: "nps copy.src copy.props",
-			src: `node "${LIB}/copy-and-watch/index.js" --silent "src/**/*.{js,json}" dist/`,
+			default: "nps copy.props",
```

Comment:
just remove copy.default and copy.props and put the props command as copy, there is no need to have a copy.default that calls copy.props

Reply:
Done